src/components/form/review-form.tsx

```diff
@@ -117,9 +119,10 @@ export const benefits = [
   { field: "freeMerch", label: "Free merchandise" },
 ];
 
-const steps = [
+const steps: string | any[] = [
```

Comment:
Does the type `{fields: string[], color: string}[]` work? I know you mentioned this being weird but I can't remember why